include/swift/AST/Stmt.h

```diff
@@ -379,6 +385,8 @@ class alignas(8) PoundAvailableInfo final :
   void setVariantAvailableRange(const VersionRange &Range) {
     VariantAvailableRange = Range;
   }
+
+  bool getIsUnavailability() const { return isUnavailability; }
```

Comment:
I think it would be better to rename the method to `isUnavailability` and rename the field to `_isUnavailability` (or something else). Predicate methods should generally be named `isXYZ()`.